specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -681,6 +789,9 @@
               "$ref": "#/definitions/ErrorResponse"
             }
           }
+        },
+        "x-ms-pageable": {
```

Comment:
add 'x-ms-pagable' header to existing tag will cause sdk breaking change.

Reply:
confirmed with @wilcobmsft , this change is requested from ARM team, and currently no SDK released for it,  and customers using this version API is limited known one. it's safe to make the change.